src/openloco/map/tilemgr.cpp

```diff
@@ -2,17 +2,24 @@
 #include "../input.h"
 #include "../interop/interop.hpp"
 #include "../viewportmgr.h"
+#include <cassert>
 
 using namespace openloco::interop;
 
 namespace openloco::map::tilemgr
 {
+    constexpr uint16_t MAP_SELECT_FLAG_ENABLE = 1;
+    constexpr uint16_t MAP_SELECT_FLAG_ENABLE_CONSTRUCT = 2;
```

Comment:
We usually define flags using enums, e.g.:
```suggestion
    enum MapSelectFlag : uint16_t
    {
        enable = (1 << 0),
        enableConstruct = (1 << 1),
    }
```

Reply:
Implemented in the next commit.